### DISCUSSION TITLE
Is Pekko affected by CVE-2023-45865?

### DISCUSSION THREAD

**Discussion by slandelle** [apache/pekko#851] [General]
Hi,

Sorry if here is not the right place to ask. If not, please guide me.

[CVE-2023-45865](https://akka.io/security/akka-cve-2023-45865.html) hits all Akka versions up to 2.8.5 included, so I guess Pekko is affected as well.

Was the Pekko project alerted too? Is there a planned fix?

Best regards,

Stéphane

**Comment by pjfanning** [comment]
see https://github.com/apache/incubator-pekko/pull/771 - included in pekko 1.0.2 release

**Comment by slandelle** [comment]
Oh, thanks a lot.
I'm very sorry for the noise.
